packages/angular-cli/scripts/install.ts

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+import {stripIndent} from 'common-tags';
+import {yellow} from 'chalk';
+
+const SemVer = require('semver').SemVer;
+
+const version = new SemVer(process.version);
+if (version.compare('6.9.0') < 0) {
+  console.log(yellow(stripIndent`
+    We detected that you are using Node v${version.version}. Unfortunately, this version will not
+    be officially supported when the Angular CLI is released. The official Node version that will
+    be supported is 6.9 and greater. This beta release (Beta.27) will be the last release to
+    support Node 4.
+
+    This is to ensure that we can provide our users with better support for the upcoming releases.
+    Many of our dependencies are moving to deprecating Node 4 already and that would mean that we
+    could not update them to the latest versions which might have bug fixes and new useful features.
+
+    As a forewarning, We are also moving to "@angular/cli" with the next release, which will only
```

Comment:
Good point.  We're still trying to figure out where to put those warnings, since post install might be too early and be missed out when looking at the console.